llpc/test/shaderdb/gfx11/SgprUserDataInit_Fs.pipe

```diff
@@ -0,0 +1,431 @@
+; CHECK-NEXT:    s_cbranch_execz .LBB0_4
+; CHECK-NEXT:    v_mov_b32_e32 v0, 1.0
+; CHECK-NEXT:    v_mov_b32_e32 v1, 0
+; CHECK-NEXT:    exp pos0 v1, v1, v1, v0 done
+; CHECK-NEXT:  .LBB0_4:
+; CHECK-NEXT:    s_endpgm
+;
+; CHECK-LABEL: amdgpu_ps_main:
+; CHECK:         s_getpc_b64 s[0:1]
+; CHECK-NEXT:    s_mov_b32 s8, exec_lo
+; CHECK-NEXT:    s_mov_b32 s3, s1
+; CHECK-NEXT:    s_wqm_b32 exec_lo, exec_lo
+; CHECK-NEXT:    s_load_b256 s[0:7], s[2:3], 0x180
+; CHECK-NEXT:    v_mov_b32_e32 v0, 0x58005800
+; CHECK-NEXT:    s_and_b32 exec_lo, exec_lo, s8
+; CHECK-NEXT:    s_waitcnt lgkmcnt(0)
+; CHECK-NEXT:    image_sample v[0:3], v0, s[0:7], s[0:3] dmask:0xf dim:SQ_RSRC_IMG_2D a16
+; CHECK-NEXT:    s_waitcnt vmcnt(0)
+; CHECK-NEXT:    v_dual_mul_f32 v0, v0, v0 :: v_dual_mul_f32 v1, v1, v1
+; CHECK-NEXT:    v_dual_mul_f32 v2, v2, v2 :: v_dual_mul_f32 v3, v3, v3
+; CHECK-NEXT:    s_delay_alu instid0(VALU_DEP_2) | instskip(NEXT) | instid1(VALU_DEP_2)
+; CHECK-NEXT:    v_cvt_pk_rtz_f16_f32_e32 v0, v0, v1
+; CHECK-NEXT:    v_cvt_pk_rtz_f16_f32_e32 v1, v2, v3
+; CHECK-NEXT:    exp mrt0 v0, v1, off, off done
+; CHECK-NEXT:    s_endpgm
+;
+; CHECK-LABEL: .amdgpu_pal_metadata
+; CHECK-NEXT: ---
+; CHECK-NEXT: amdpal.pipelines:
+; CHECK-NEXT:   - .api:            Vulkan
+; CHECK-NEXT:     .graphics_registers:
+; CHECK-NEXT:       .aa_coverage_to_shader_select: InputCoverage
+; CHECK-NEXT:       .cb_shader_mask:
+; CHECK-NEXT:         .output0_enable: 0xf
+; CHECK-NEXT:         .output1_enable: 0
+; CHECK-NEXT:         .output2_enable: 0
+; CHECK-NEXT:         .output3_enable: 0
+; CHECK-NEXT:         .output4_enable: 0
+; CHECK-NEXT:         .output5_enable: 0
+; CHECK-NEXT:         .output6_enable: 0
+; CHECK-NEXT:         .output7_enable: 0
+; CHECK-NEXT:       .db_shader_control:
+; CHECK-NEXT:         .alpha_to_mask_disable: true
+; CHECK-NEXT:         .conservative_z_export: 0
+; CHECK-NEXT:         .depth_before_shader: 0
+; CHECK-NEXT:         .exec_on_hier_fail: false
+; CHECK-NEXT:         .exec_on_noop:   false
+; CHECK-NEXT:         .kill_enable:    false
+; CHECK-NEXT:         .mask_export_enable: false
+; CHECK-NEXT:         .pre_shader_depth_coverage_enable: 0
+; CHECK-NEXT:         .stencil_test_val_export_enable: 0
+; CHECK-NEXT:         .z_export_enable: 0
+; CHECK-NEXT:         .z_order:        0x1
+; CHECK-NEXT:       .es_vgpr_comp_cnt: 0
+; CHECK-NEXT:       .ge_ngg_subgrp_cntl:
+; CHECK-NEXT:         .prim_amp_factor: 0x1
+; CHECK-NEXT:         .threads_per_subgroup: 0x100
+; CHECK-NEXT:       .gs_vgpr_comp_cnt: 0x1
+; CHECK-NEXT:       .ia_multi_vgt_param_piped:
+; CHECK-NEXT:         .primgroup_size: 0x7f
+; CHECK-NEXT:       .max_verts_per_subgroup: 0x80
+; CHECK-NEXT:       .pa_cl_clip_cntl:
+; CHECK-NEXT:         .dx_linear_attr_clip_ena: true
+; CHECK-NEXT:         .rasterization_kill: false
+; CHECK-NEXT:         .vte_vport_provoke_disable: false
+; CHECK-NEXT:       .pa_cl_vs_out_cntl: {}
```

Comment:
@WenqingLiAMD The failure of SgprUserDataInit_Fs.pipe can be fixed by removing line 185 : `pa_cl_vs_out_cntl: {}`. The fix is made in porting https://github.amd.com/AMD-Radeon-Driver/llpc/pull/2506.